bokehjs/src/coffee/document.coffee

```diff
@@ -148,6 +152,23 @@ export class Document
     finally
       @_pop_all_models_freeze()
 
+  interactive_start: (plot) ->
+    if @_interactive_plot == null
+      @_interactive_plot = plot
+      @_interactive_plot.trigger_event(new LODStart({}))
+    @_interactive_timestamp = Date.now()
+
+  interactive_stop: (plot) ->
+    if @_interactive_plot?.id == plot.id
```

Comment:
@bryevdv, what's the reason for comparing ids instead of just references (`@_interactive_plot == plot`)?